### PR TITLE
ixgbe: Fix typos

### PR DESCRIPTION
**Pull request by IanMarcoMoffett**

### CODE REVIEW

**Comment by emaste** [comment]
image
"committed in 3 hours" -> FYI it seems your clock is set incorrectly

**Comment by IanMarcoMoffett** [comment]
> image
> 
> "committed in 3 hours" -> FYI it seems your clock is set incorrectly

I didn't realize that, thank you.

**Comment by IanMarcoMoffett** [comment]
I believe I fixed the date.

File: sys/dev/ixgbe/ixgbe_api.c [hunk 1]
```diff
@@ -1420,8 +1420,8 @@ s32 ixgbe_bypass_rw(struct ixgbe_hw *hw, u32 cmd, u32 *status)
  * ixgbe_bypass_valid_rd - Verify valid return from bit-bang.
  *
  * If we send a write we can't be sure it took until we can read back
- * that same register.  It can be a problem as some of the feilds may
- * for valid reasons change inbetween the time wrote the register and
```
Comment (gmshake) [inline sys/dev/ixgbe/ixgbe_api.c:1424]: I think the word 'inbetween' is intended. Some people may favor 'in-between' .

**Review by gmshake — COMMENTED** [review]
See in-line comment

**Comment by IanMarcoMoffett** [comment]
No way my cat just jumped on my computer and marked it as resolved. Sorry

**Comment by gmshake** [comment]
Hi @IanMarcoMoffett ,
The word `feilds` is apparently wrong and should be `fields`.
What's your preferred full name and/or email (will be in commit log and public)? I'd like commit part of this PR and give you credits.

**Comment by IanMarcoMoffett** [comment]
> Hi @IanMarcoMoffett ,
> 
> The word `feilds` is apparently wrong and should be `fields`.
> 
> What's your preferred full name and/or email (will be in commit log and public)? I'd like commit part of this PR and give you credits.
> 
> 
> 
> 

Ian Moffett
ian@mkall.org

**Comment by gmshake** [comment]
Committed in 99cb088f1a25 .

**Comment by IanMarcoMoffett** [comment]
> Committed in 99cb088f1a25 .

Thanks!!! 👍

**Pull request closed without merging**